src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx

```diff
@@ -73,7 +73,42 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
   const [isDateRangeInvalid, setIsDateRangeInvalid] = useState(false);
   const [isQueryEditorFocused, setIsQueryEditorFocused] = useState(false);
   const opensearchDashboards = useOpenSearchDashboards<IDataPluginServices>();
-  const { uiSettings, storage, appName, data } = opensearchDashboards.services;
+  const { uiSettings, storage, appName, data, keyboardShortcut } = opensearchDashboards.services;
+
+  keyboardShortcut?.useKeyboardShortcut({
+    id: 'open_date_picker',
+    pluginId: 'data',
+    name: 'Open Date Picker',
+    category: 'Open',
+    keys: 'shift+t',
+    execute: () => {
+      const selectors = [
+        '[data-test-subj="superDatePickerstartDatePopoverButton"]',
```

Comment:
Out of scope for this PR, but we should keep these in a constants file in case it gets updated in the future.